src/test/java/nextstep/subway/unit/LineServiceTest.java

```diff
@@ -4,20 +4,29 @@
 import nextstep.subway.line.repository.LineRepository;
 import nextstep.subway.line.service.LineService;
 import nextstep.subway.section.dto.CreateSectionRequest;
+import nextstep.subway.section.policy.add.AddSectionPolicy;
+import nextstep.subway.section.repository.Section;
+import nextstep.subway.section.repository.Sections;
 import nextstep.subway.station.service.StationFindable;
 import nextstep.subway.unit.fake.FakeLineRepository;
 import nextstep.subway.unit.fake.FakeStationFindable;
 import org.junit.jupiter.api.Test;
 
-import static nextstep.subway.unit.fixture.LineFixture.신분당선_ID;
-import static nextstep.subway.unit.fixture.StationFixture.논현역_ID;
-import static nextstep.subway.unit.fixture.StationFixture.신논현역_ID;
+import static nextstep.fixture.LineFixture.신분당선_ID;
+import static nextstep.fixture.StationFixture.논현역_ID;
+import static nextstep.fixture.StationFixture.신논현역_ID;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LineServiceTest {
     private final LineRepository lineRepository = new FakeLineRepository();
     private final StationFindable stationFindable = new FakeStationFindable();
-    private final LineService lineService = new LineService(lineRepository, stationFindable);
+    private final AddSectionPolicy addSectionPolicy = new AddSectionPolicy() {
+        @Override
+        public void validate(Sections sections, Section section) {
+            // nothing
+        }
+    };
+    private final LineService lineService = new LineService(lineRepository, stationFindable, addSectionPolicy);
```

Comment:
```suggestion
    private final LineService lineService = new LineService(lineRepository, stationFindable, (sections, section) -> {});
```

AddSectionPolicy 요소를 따로 검증했기 때문에 검증이 필요없다고 생각하신 것 같은데, 위와 같이 간단하게 할수도 있습니다 😅